test/utils.test.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-var Buffer = require('safe-buffer').Buffer;
-var utils = require('../lib/utils');
-var assert = require('assert');
-var debug = require('debug');
+const Buffer = require('safe-buffer').Buffer;
```

Comment:
Still using safe buffer here

Reply:
Thanks. This still works because mocha pulls in safe-buffer anyway, but we should fix that.